src/chains/filecoin/options/src/wallet-options.ts

```diff
@@ -66,7 +67,13 @@ export type WalletConfig = {
 
 export const WalletOptions: Definitions<WalletConfig> = {
   totalAccounts: {
-    normalize,
+    normalize: rawInput => {
+      if (rawInput >= MAX_ACCOUNTS)
+        console.log(
+          `wallet.totalAccounts set above MAX_ACCOUNTS: ${MAX_ACCOUNTS} and may affect performance.`
+        );
+      return rawInput;
+    },
```

Comment:
Let's not change filecoin's settings.

Reply:
:+1: https://github.com/trufflesuite/ganache/pull/3361/commits/11bdfdb0aa0de9fd1a50f3291a77b0bde982b68a